docs/howto/use-alternative-secrets-backend.rst

```diff
@@ -117,7 +118,10 @@ The following is a sample JSON file.
     }
 
 The YAML file structure is similar to that of a JSON. The key-value pair of connection ID and the definitions of one or more connections.
-The connection can be defined as a URI (string) or JSON object.
+The connection can be defined as a URI (string) or JSON object. Any extra json parameters can be provided using keys like ``extra_dejson`` and ``extra``.
```

Comment:
What do you think to put it earlier in the text?

> The file can be defined in ``JSON``, ``YAML`` or ``env`` format.  The ``JSON`` and ``YAML`` formats support defining the connection as an object or as URI (string). The env format supports only URI (string).  For a description of the connection object parameters see :class:`~airflow.models.connection.Connection`. We additionally support ``extra_dejson`` to allows you to define extra parameters as object where as the key ``extra`` can be used in case of a JSON string.
The keys ``extra`` and ``extra_dejson`` are mutually exclusive.

Reply:
Makes sense since those keys are common not just specific to YAML.
Thanks. Updated !